src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -285,41 +303,83 @@ def _get_job_states_by_cell_id(self, req: JobRequest = None) -> dict:
 
     def _get_job_info(self, req: JobRequest) -> dict:
         """
-        Look up job info. This is just some high-level generic information about the running
-        job, including the app id, name, and job parameters.
-        :param req: a JobRequest with the job_id_list of interest
-        :returns: a dict keyed with job IDs and with values of dicts with the following keys:
-            - app_id - str - module/name,
-            - app_name - str - name of the app as it shows up in the Narrative interface
-            - batch_id - str - the batch parent ID (if appropriate)
-            - job_id - str - just re-reporting the id string
-            - job_params - dict - the params that were passed to that particular job
+        Gets job information for a list of job IDs.
+
+        Job info for a given job ID is in the form:
+        {
+            "app_id": string in the form "<module>/<name>",
+            "app_name": string,
+            "job_id": string,
+            "job_params": dictionary,
+            "batch_id": string | None,
+        }
+
+        Jobs that cannot be found in the `_running_jobs` index will return
+        {
+            "job_id": string,
+            "error": "Cannot find job with ID <job_id>"
+        }
+
+        :param req: job request with a list of job IDs
+        :type req: JobRequest
+        :return: dictionary containing job info for each input job, indexed by job ID
+        :rtype: dict
         """
         job_id_list = self._get_job_ids(req)
         job_info = self._jm.get_job_info(job_id_list)
         self.send_comm_message(MESSAGE_TYPE["INFO"], job_info)
         return job_info
 
-    def __get_job_states(self, job_id_list) -> dict:
```

Comment:
If being totally pedantic the leading dunderscore is for name mangling in times of subclassing

Reply:
can you give this method a better name than `_get_send_job_states`, in that case? e.g. `_get_job_states_from_job_id_list`?